assumptions/tools/build.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env sh
+set -ex
+
+cd "${0%/*}"
+virtualenv -p python .virtualenv
```

Comment:
I'm not sure I like all these things creating their own virtualenv everywhere. We have a python package for interacting with a project-wide virtualenv, maybe we should use that instead of adding random shell scripts.